### PR TITLE
Create empty restartless add-on shell

### PR DESCRIPTION
**Pull request by jaredhirsch**
@chuckharmston R? Not a ton to look at here, but happy to answer questions or add code docs if anything's unclear.

### CODE REVIEW

File: lib/universal-search.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const EXPORTED_SYMBOLS = ['UniversalSearch'];
+
+function UniversalSearchCtor() {}
```
Comment (chuckharmston) [inline lib/universal-search.js]: Mind using the full word `Constructor`?

**Comment by chuckharmston** [comment]
Might also want to add "(closes #2, closes #19)" in the commit message. Feel free to merge!

File: lib/universal-search.js (hunk 1 above)
Comment (jaredhirsch) [inline lib/universal-search.js]: Sure, sounds good

Comment (jaredhirsch) [inline lib/universal-search.js]: It looked too Java-ish, so I decided to just call the constructor 'Search' ¯_(ツ)_/¯

**Comment by jaredhirsch** [comment]
Thanks, @chuckharmston! Merging.

**Pull request merged**